pkgs/applications/networking/instant-messengers/telegram/tdesktop/default.nix

```diff
@@ -20,27 +20,39 @@ with lib;
 
 let
   tg_owt = callPackage ./tg_owt.nix {};
-  tgcalls-gcc10-fix = fetchpatch { # "Fix build on GCC 10, second attempt."
-    url = "https://github.com/TelegramMessenger/tgcalls/commit/eded7cc540123eaf26361958b9a61c65cb2f7cfc.patch";
-    sha256 = "19n1hvn44pp01zc90g93vq2bcr2gdnscaj5il9f82klgh4llvjli";
+  webviewPatch = writeTextFile {
```

Comment:
Any reason to use `writeTextFile` here? Usually we just add patches as files or use `fetchpatch`. The latter is actually preferred: https://nixos.org/manual/nixpkgs/unstable/#sec-patches

So in this case I'd actually fetch https://github.com/archlinux/svntogit-community/blob/013eff77a13b6c2629a04e07a4d09dbe60c8ca48/trunk/fix-webview-includes.patch.

Upstream link (but irrelevant as not yet merged): https://github.com/desktop-app/lib_webview/pull/1

Reply:
No reason, just didn't realise there was a git mirror of the Arch community repo